addon/snapshot.js

```diff
@@ -1,6 +1,10 @@
 import Ember from 'ember';
 import { getNativeXhr } from './native-xhr';
 import { maybeDisableMockjax, maybeResetMockjax } from './mockjax-wrapper';
+import $ from 'jquery';
```

Comment:
Whoa.

Reply:
😄 

If you're interested in updating `ember-cli-babel` to version 6.3+ (I believe) you could also use the new Ember imports outlined here: https://github.com/ember-cli/ember-rfc176-data but that would likely be appropriate for another PR. There are tools to automate the change as well. Let me know if you're interested!